src/components/templates/PartyMap/components/RoomModal/RoomModal.tsx

```diff
@@ -102,11 +106,7 @@ export const RoomModalContent: React.FC<RoomModalContentProps> = ({
       )}
 
       <div className="room-modal__main">
-        {room.image_url ? (
-          <img src={room.image_url} alt={room.title} />
-        ) : (
-          <span>{room.title}</span>
-        )}
+        <div className="room-modal__icon" style={iconStyle}></div>
```

Comment:
When there is no 'body' to a `div`/etc, it's better to use the shorthand format rather than having an explicit open/closing tag:

```suggestion
        <div className="room-modal__icon" style={iconStyle} />
```